frontend/ui/widget/dictquicklookup.lua

```diff
@@ -185,25 +254,36 @@ function DictQuickLookup:update()
             },
             {
                 {
-                    text = _("Wikipedia"),
-                    enabled = not self.wiki,
+                    text = self.wiki and _("Full page") or _("Wikipedia"),
```

Comment:
I'll do that.
(But no, here, i was using self.wiki to distinguish wiki from dict : if wiki, this button is "Full Page" - if dict, this button is "Wikipedia" like before. We're already in the "not fullpage" buttons branch - but ok, will be easier to read without these x and y or z)